### PR TITLE
New package: hid_bootloader_cli

### PR DESCRIPTION
**Pull request by fauxpark**
This is a modified version of teensy_loader_cli (#6685) which also supports devices with LUFA's HID bootloader implementation.

### CODE REVIEW

**Comment by Biswa96** [comment]
The patch step should be in prepare function.

**Comment by fauxpark** [comment]
Should I also fix this in the teensy_loader_cli package?

File: mingw-w64-hid-bootloader-cli/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+_realname=hid_bootloader_cli
+pkgbase=mingw-w64-${_realname//_/-}
+pkgname=${MINGW_PACKAGE_PREFIX}-${_realname//_/-}
+pkgver=210130
+pkgrel=1
+pkgdesc='LUFA HID Bootloader CLI (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('GPL3')
```
Comment (lazka) [inline mingw-w64-hid-bootloader-cli/PKGBUILD:11]:
```suggestion
license=('custom:MIT')
```

See https://fedoraproject.org/wiki/Licensing:MIT#Old_Style_with_legal_disclaimer_2 https://github.com/abcminiuser/lufa/blob/master/LUFA/License.txt

Comment (fauxpark) [reply mingw-w64-hid-bootloader-cli/PKGBUILD:11]: The HID Bootloader CLI itself seems to be GPL3, not MIT: https://github.com/abcminiuser/lufa/tree/master/Bootloaders/HID/HostLoaderApp
Which makes sense as teensy_loader_cli is GPL3: https://github.com/paulstoffregen/teensy_loader_cli

Comment (lazka) [reply mingw-w64-hid-bootloader-cli/PKGBUILD:11]: I see, thanks

**Pull request merged**